build/linux/installer/conf/fluent-bit-geneva-logs_tenant.conf

```diff
@@ -29,3 +29,16 @@
     Workers                         ${OUTPUT_FORWARD_WORKERS_COUNT}
     Retry_Limit                     ${OUTPUT_FORWARD_RETRY_LIMIT}
     Require_ack_response            ${REQUIRE_ACK_RESPONSE}
+
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               geneva.container.log.*
```

Comment:
should be geneva.container.log.tenant.*


also move this between input and output

Reply:
changed